README.md

```diff
@@ -147,6 +147,8 @@ operating systems to RISC-V.
     - Demonstrates usage of the PWM API to generate waveforms.
   - mem-latency
     - A memory test that measure the latency at different cache layers and memory blocks
+  - example-hardware-performance-monitor
+    - Demonstrates usage of the Risc-V hardware performance counter APIs.
```

Comment:
Miss one, please change as follows:
- example-hpm
   - Demonstrates usage of the RISC-V hardware performance monitor counter APIs.